dimensionality_reduction_api/Dockerfile

```diff
@@ -10,9 +10,10 @@ RUN apt-get -y update && \
     adduser --no-create-home --shell /bin/false --disabled-password --uid 1001 --system --group app && \
     pip install --upgrade -r requirements.txt --no-cache-dir && \
     rm requirements.txt
+RUN  pip install Pillow
```

Comment:
Here that stuck to us because we were using it to manipulate images and later we saw that we could solve it with PIL.
Also, at that time I had not realized that the libraries could be installed by putting them in the requirements.txt file.